packages/demo-html/public/widget-js.html

```diff
@@ -16,7 +16,12 @@
     <div id="wrapper"></div>
     <script src="./lib/embed-next.js"></script>
     <script>
-      window.tf.createWidget("moe6aa", { container: document.getElementById("wrapper"), medium: "unit-test" });
+      window.tf.createWidget("moe6aa", {
+        container: document.getElementById("wrapper"),
+        source: "localhost",
+        transferableUrlParameters: ["foo"],
```

Comment:
Should be `transitiveSearchParams`, please include 2 types here to demonstrate array types work (`"foo", "bar"` like in `widget-html.html`).